app/assets/stylesheets/recipeIndex.css

```diff
@@ -0,0 +1,97 @@
+#recipes-holder {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  width: 100%;
+  height: 100%;
+}
+
+#recipes {
```

Comment:
- [ ] Let's not use ids as selectors in your code as it makes the code dry.

Reply:
Following commit for the solution:

<img width="551" alt="image" src="https://user-images.githubusercontent.com/53873573/226094769-e0f44b64-cd92-4c4d-8d70-d3dd96d97f27.png">